src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -99,7 +99,7 @@ tnsr::a<DataType, SpatialDim, Frame> gauge_source(
         trace_christoffel_last_indices) noexcept {
   DataType one_over_lapse = 1.0 / get(lapse);
   auto gauge_source_h =
-      make_with_value<tnsr::a<DataType, SpatialDim, Frame>>(lapse, 0.0);
+      make_with_value<tnsr::a<DataType, SpatialDim, Frame>>(lapse, 0.);
```

Comment:
Why this change?

Reply:
I'm guissing this is to make the `0.0` and `0.` consistent in the file.

Reply:
Unfortunately the file wasn't actually consistent beforehand... and still isn't now.

Reply:
It should be consistent now with the use of `0.` instead of `0.0`.